### PR TITLE
New package: clipmenu-5.6.0

### PR DESCRIPTION
**Pull request by hristoast**

### CODE REVIEW

File: srcpkgs/clipmenu/template [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Template file for 'clipmenu'
+pkgname=clipmenu
+version=5.6.0
+revision=1
+noarch=yes
+build_style=fetch
```
Comment (Chocimier) [inline srcpkgs/clipmenu/template]: Remove `build_style=` line. Files will be extracted into current dir of do_install.

File: srcpkgs/clipmenu/template [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# Template file for 'clipmenu'
+pkgname=clipmenu
+version=5.6.0
+revision=1
+noarch=yes
+depends="bash"
+create_wrksrc=yes
+short_desc="Clipboard management using dmenu"
+maintainer="Hristos N. Triantafillou <clipmenu@hristos.lol>"
```
Comment (maxice8) [inline srcpkgs/clipmenu/template]: Please don't have a different mail for each package

File: srcpkgs/clipmenu/template [hunk 3]
```diff
@@ -0,0 +1,20 @@
+# Template file for 'clipmenu'
+pkgname=clipmenu
+version=5.6.0
+revision=1
+noarch=yes
+create_wrksrc=yes
```
Comment (Chocimier) [inline srcpkgs/clipmenu/template]: Without `create_wrksrc=yes`, `$pkgname-$version/` in do_install will not be needed.

**Comment by hristoast** [comment]
I think it might be good to add a post-install message that recommends `clipnotify`, and `dmenu` or `rofi`.  I'll work on this.

**Pull request merged**